src/hooks/useOperatorTrades.ts

```diff
@@ -72,44 +48,77 @@ export function useOrderTrades(order: Order | null): Result {
   const [isLoading, setIsLoading] = useState(false)
   const [error, setError] = useState<UiError>()
   const [trades, setTrades] = useState<Trade[]>([])
+  const [rawTrades, setRawTrades] = useState<RawTrade[]>([])
+  const [tradesTimestamps, setTradesTimestamps] = useState<TradesTimestamps>({})
 
   // Here we assume that we are already in the right network
   // contrary to useOrder hook, where it searches all networks for a given orderId
   const networkId = useNetworkId()
 
   const fetchTrades = useCallback(
-    async (controller: AbortController, _networkId: Network, _order: Order): Promise<void> => {
+    async (controller: AbortController, _networkId: Network): Promise<void> => {
+      if (!order) return
+
       setIsLoading(true)
 
-      const { uid: orderId, buyToken, sellToken } = _order
+      const { uid: orderId } = order
 
       try {
-        const _trades = await getTrades({ networkId: _networkId, orderId })
+        const trades = await getTrades({ networkId: _networkId, orderId  })
 
         if (controller.signal.aborted) return
 
-        setTrades(_trades.map((trade) => ({ ...transformTrade(trade), buyToken, sellToken })))
+        setRawTrades(trades)
         setError(undefined)
       } catch (e) {
         const msg = `Failed to fetch trades`
         console.error(msg, e)
+
         setError({ message: msg, type: 'error' })
       } finally {
         setIsLoading(false)
       }
     },
-    [],
+    [order],
   )
 
+  // Fetch blocks timestamps for trades
+  useEffect(() => {
+    setTradesTimestamps({})
+
+    fetchTradesTimestamps(rawTrades).then(setTradesTimestamps).catch(error => {
```

Comment:
same issue with linter here 
<img width="736" alt="image" src="https://user-images.githubusercontent.com/2352112/230350090-941f7b4f-b6fd-4c26-8b99-79059c2e918f.png">

mine likes it like this, which i think is nicer in this case 

<img width="600" alt="image" src="https://user-images.githubusercontent.com/2352112/230350172-458e6073-c1cf-462e-9aaf-9d4847d33b81.png">

Reply:
Fixed, and added `.editorconfig`